packages/react-aria-components/stories/Calendar.stories.tsx

```diff
@@ -74,53 +74,61 @@ export const CalendarResetValue: CalendarStory = {
   )
 };
 
-function CalendarMultiMonthExample(args) {
-  let defaultDate = new CalendarDate(2021, 7, 1);
-  let [focusedDate, setFocusedDate] = React.useState(defaultDate);
-
-  return (
-    <>
```

Comment:
did you mean to delete the stuff related to `selectionAlignment`? can we leave this story as it was or was there a reason for changing it?

Reply:
I'll fix that on Monday, thank you